graalpy-notebook-example/pom.xml

```diff
@@ -128,9 +144,8 @@
                         <configuration>
                             <rules>
                                 <requireFilesExist>
-                                    <message>You must set JAVA_HOME to a GraalVM with Python installed.</message>
                                     <files>
-                                        <file>${env.JAVA_HOME}/bin/graalpy</file>
```

Comment:
Why do we need to change the env name?

Reply:
It doesn’t change the `env` name. The name of env is here `<argument>venv</argument>`.